modulefiles/ufs_hera.gnu

```diff
@@ -18,25 +18,7 @@ setenv CMAKE_Platform hera.gnu
 module use /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/modulefiles/stack
 module load hpc/1.1.0
 module load hpc-gnu/9.2.0
-module load hpc-mpich/3.3.2  
+module load hpc-mpich/3.3.2
```

Comment:
is `hpc-mpich/3.3.2` meta-module not loading `mpich/3.3.2`? If so, the next line should be removed.